src/resolvers.ts

```diff
@@ -20,6 +22,15 @@ export const resolvers = {
       return { highlights };
     },
   },
+  Highlight: {
+    note: async (
```

Comment:
I smell a data loader in the near future

Reply:
![smell](https://user-images.githubusercontent.com/1273879/154151665-7458ee7a-8b0a-4b9b-925e-70f1493de064.gif)

Reply:
yep